test/Microsoft.ML.AutoML.Tests/ColumnInferenceTests.cs

```diff
@@ -186,5 +187,18 @@ public void InferColumnsColumnInfoParam()
             Assert.Equal(DefaultColumnNames.Features, result.ColumnInformation.NumericColumnNames.First());
             Assert.Null(result.ColumnInformation.ExampleWeightColumnName);
         }
+
+        [Fact]
+        public void TrySplitColumns_should_split_on_dataset_with_newline_between_double_quotes()
+        {
```

Comment:
_(pinned comment to code to allow threaded discussion)_

The original issue for this (#4460) mentioned problems with using files with new line characters inside quoted fields, when using ColumnInference() in this way:
```
MLContext mlContext = new MLContext();
var inputColumnInformation = new ColumnInformation();
inputColumnInformation.LabelColumnName = @"review_scores_rating";
var train = mlContext.Auto().InferColumns(TrainDataPath, inputColumnInformation);
```
So please, add a test actually testing this scenario of calling InferColumns() directly. I was under the impression that to fix this scenario other changes needed to be done in the ColumnInference internal API:
https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.AutoML/ColumnInference/ColumnInferenceApi.cs

Reply:
### Resolved